core/versionutil.go

```diff
@@ -96,7 +96,7 @@ func FetchMavenVersionPrefixedList(url string, friendlyName string) func(mcVersi
 		}
 		// Sort list to get largest version
 		flexver.VersionSlice(out.Versioning.Versions.Version).Sort()
-		return allowedVersions, allowedVersions[len(allowedVersions)-1], nil
+		return allowedVersions, allowedVersions[0], nil
```

Comment:
~~What's the reason for this change? The second return value should be the newest version (i.e. the last version when sorted)~~ Discussed on discord - the previous line needs to be fixed, whether you want to do so in this PR or push it separately.